tools/DockerTestServer/gamecfg/game_options.txt

```diff
@@ -0,0 +1,471 @@
+## HEALTH ###
```

Comment:
Why are you including all of game_options.txt? Can we make this a symlink or similar? I don't want to need both maintained.

Reply:
You could also keep a file *called* game_options.txt, then inside write a comment saying like "If you type anything in here, it will override the codebase game_options.txt", maybe